gnocchi/carbonara.py

```diff
@@ -407,6 +410,10 @@ def __init__(self, value, sampling):
             self.key = value.key
         elif isinstance(value, pandas.Timestamp):
             self.key = value.value / 10e8
+        elif isinstance(value, numpy.datetime64):
```

Comment:
if i remove this, the only tests that fail are the ones changed as part of this commit. i get the feeling this isn't needed since it's remove in your later in your chain of commits.